main.go

```diff
@@ -36,7 +36,19 @@ var Flags = struct {
 		Username    string
 		Password    string
 		BearerToken string
-		Insecure    bool
+		TLS         struct {
+			Authority struct {
+				Certificate     string
+				CertificateFile string
+			}
+			Client struct {
+				Certificate     string
+				CertificateFile string
+				Key             string
+				KeyFile         string
+			}
```

Comment:
Why do we need to be able to provide both, the files and the pem encoded content? We tent to only provide the files, because `certctl` (or in the future some cert operator) writes files. 

Check how it is done in Kubernetesd: https://github.com/giantswarm/kubernetesd/blob/master/main.go#L182-L185. 

See also how we create the kubernetes client in kubernetesd: https://github.com/giantswarm/kubernetesd/blob/master/service/service.go#L88.